cdci_data_analysis/flask_app/app.py

```diff
@@ -259,6 +259,13 @@ def inspect_state():
     return state_data_obj
 
 
+# TODO perhaps the way we name the functionality should be better phrased
+@app.route('/get-user-specific-instrument-list')
```

Comment:
it should rather be user-specific by default. We also want to actively prevent users from accessing other instruments.

I.e. the regular old endpoint name would work

Reply:
renamed. Also once this will be merged we will adapt oda_api to use this endpoint so we can safely remove the old endpoint